components/picker/index.css

```diff
@@ -10,409 +10,532 @@ OF ANY KIND, either express or implied. See the License for the specific languag
+    --highcontrast-picker-border-color-active: ButtonText;
+    --highcontrast-picker-border-color-key-focus: Highlight;
 
-  /* todo: use real DNA var */
-  --spectrum-picker-texticon-border-size-increase-focus: 1px;
+    --highcontrast-picker-border-color-error-default-open: ButtonText;
+    --highcontrast-picker-border-color-error-hover: ButtonText;
+    --highcontrast-picker-border-color-error-active: ButtonText;
 
-  /* Adjustments for inset/outset padding in DNA */
-  --spectrum-picker-padding-left-adjusted: calc(var(--spectrum-picker-texticon-padding-left) - var(--spectrum-picker-texticon-border-size));
-  --spectrum-picker-padding-right-adjusted: calc(var(--spectrum-picker-texticon-padding-right) - var(--spectrum-picker-texticon-border-size));
-  --spectrum-picker-textonly-padding-left-adjusted: calc(var(--spectrum-picker-textonly-padding-left) - var(--spectrum-picker-textonly-border-size));
-  --spectrum-picker-textonly-padding-right-adjusted: calc(var(--spectrum-picker-textonly-padding-right) - var(--spectrum-picker-textonly-border-size));
-  --spectrum-picker-focus-ring-border-radius-adjusted: calc(var(--spectrum-picker-textonly-border-radius) + var(--spectrum-picker-focus-ring-gap));
-}
+    --highcontrast-picker-font-color-default: ButtonText;
+    --highcontrast-picker-font-color-default-open: ButtonText;
+    --highcontrast-picker-font-color-key-focus: ButtonText;
+    --highcontrast-picker-font-color-disabled: GrayText;
 
-/* special cases for focus-ring */
-.spectrum-Picker {
-  --spectrum-picker-focus-ring-gap: var(--spectrum-alias-component-focusring-gap);
-  --spectrum-picker-focus-ring-size: var(--spectrum-alias-component-focusring-size);
-  --spectrum-picker-focus-ring-color: var(--spectrum-picker-m-textonly-focusring-border-color-key-focus);
+    --highcontrast-picker-background-color-default: Background;
+    --highcontrast-picker-background-color-disabled: Background;
 
-  transition: border-color var(--spectrum-global-animation-duration-100) ease-in-out;
+    --highcontrast-picker-icon-color-default: ButtonText;
+    --highcontrast-picker-icon-color-default-open: ButtonText;
+    --highcontrast-picker-icon-color-hover: ButtonText;
+    --highcontrast-picker-icon-color-hover-open: ButtonText;
+    --highcontrast-picker-icon-color-key-focus: Highlight;
 
-  &:after {
-    pointer-events: none;
-    content: '';
-    position: absolute;
-    left: 0;
-    right: 0;
-    bottom: 0;
-    top: 0;
-    margin: calc((var(--spectrum-picker-focus-ring-gap) + var(--spectrum-picker-textonly-border-size)) * -1);
-    border-radius: var(--spectrum-picker-focus-ring-border-radius-adjusted);
-    transition: box-shadow var(--spectrum-global-animation-duration-100) ease-in-out;
-  }
+    --highcontrast-picker-icon-color-error-default: ButtonText;
 
-  &:focus-ring {
-    /* kill the default ring */
-    box-shadow: none;
+    --highcontrast-picker-focus-indicator-color: Highlight;
```

Comment:
Line 171 is cancelling out the correct focus indicator color on line 145. 

Incorrect (focus indicator looks blue):
![Screenshot 2023-01-13 at 1 30 51 PM](https://user-images.githubusercontent.com/25614178/212403030-07530590-13cf-4b6c-a322-6065d57514a8.png)

Correct (focus indicator looks white, component border looks blue):
![Screenshot 2023-01-13 at 1 30 36 PM](https://user-images.githubusercontent.com/25614178/212402995-df2db266-b3e3-490f-9ae5-98f0dff2275f.png)


@pfulton